.editorconfig

```diff
@@ -10,6 +10,17 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 
+# isort configuration
+[*.py]
+line_length = 79
+multi_line_output = 0
+balanced_wrapping = true
+known_future_library = future
+known_standard_library = six
+known_third_party = django, ujson, sqlalchemy, typing
+known_first_party = zerver, zproject, version, confirmation, zilencer, analytics, frontend_tests, scripts
+sections = FUTURE, STDLIB, THIRDPARTY, FIRSTPARTY, LOCALFOLDER
```

Comment:
Huh, how does this config work? It doesn't appear namespaced to `isort` -- so e.g. we're setting something called simply `line_length` to be 79 for all `*.py` files, in the `.editorconfig` which is meant to tell text editors how to format things. That sounds too broad.

Reply:
My intent would be that we don't merge this commit until we've cleaned the codebase, but it makes sense for it to hang out in PRs for sorting imports so we can talk about what config we're working towards.

Reply:
I believe that's a isort specific setting, editors aren't forced to line wrap to 79 characters.

Reply:
> My intent would be that we don't merge this commit until we've cleaned the codebase, but it makes sense for it to hang out in PRs for sorting imports

Sure, this isn't a question about the order of merging things -- it's about what we ultimately merge.

> I believe that's a isort specific setting, editors aren't forced to line wrap to 79 characters.

But there's nothing in the name of it that says that -- it's not called e.g. `isort_line_length`. It's really bad practice for isort to be introducing these generic names into a shared namespace.

It looks like isort takes a `.isort.cfg` file -- we should use that, or another alternative.

Reply:
Cool, yeah, @rht can you move this to use one of those?

Reply:
Haven't gone through all the discussion, but I am about to move the lines in .editorconfig to .isort.cfg.

Reply:
At least this is done.

Reply:
:+1: thanks!